sdk/identity/azure-identity/src/test/java/com/azure/identity/implementation/IdentityClientTests.java

```diff
@@ -454,7 +455,7 @@ private void mockForMSICodeFlow(String tokenJson) throws Exception {
         PowerMockito.doNothing().when(huc).setRequestMethod(anyString());
         PowerMockito.doNothing().when(huc).setRequestMethod(anyString());
         PowerMockito.doNothing().when(huc).setRequestMethod(anyString());
-        InputStream inputStream = new ByteArrayInputStream(tokenJson.getBytes());
+        InputStream inputStream = new ByteArrayInputStream(tokenJson.getBytes(Charset.defaultCharset()));
```

Comment:
Are tokens guaranteed to be ASCII only? If not it may be better to use `StandardCharset.UTF_8` here as the `defaultCharset` is specific to the machine. Linux and Mac will default `UTF-8` while Windows defaults `CP1252`.

Reply:
yeah, ASCII covers tokens char range.
This is test code. So, if any failures are ever encountered, it'd interesting to look at and we can fix it.